README.md

```diff
@@ -158,10 +158,6 @@ Querying data stored in Parquet, CSV, JSON, Iceberg and Delta format can be done
 	```
 
 Note, for Azure, you will need to first install the Azure extension:
```

Comment:
This line should be removed I guess?

Reply:
Actually it seems the intro should just be rewritten, because the "You may then..." sentence on its own doesn't make sense either.

Reply:
Ugh hadn't saved the file before pushing. I've pushed my last iteration, let me know